python/servo/bootstrap.py

```diff
@@ -46,8 +46,7 @@ def install_trusty_deps(force):
 
 
 def check_gstreamer_lib():
-    return subprocess.call(["pkg-config", "gstreamer-1.0 >= 1.12"],
-                           stdout=PIPE, stderr=PIPE) == 0
+    return subprocess.check_output('pkg-config --modversion gstreamer-1.0') >= 1.12
```

Comment:
Interpreting it as a float isn't a good idea, `1.2 > 1.12`

Reply:
Ah, thanks for good catch! Let me try to use another way to do so 🙇

Reply:
I think this might work? but let me try to ask those who cannot build with original command to try this one!

```
return subprocess.call(["pkg-config", "--atleast-version=1.12", "gstreamer-1.0"], stdout=PIPE, stderr=PIPE) == 0
```